src/app/pages/apps/components/installed-apps/app-info-card/app-info-card.component.ts

```diff
@@ -153,4 +155,16 @@ export class AppInfoCardComponent {
       untilDestroyed(this),
     ).subscribe();
   }
+
+  private getPortalLink(app: App, name = 'web_portal'): string {
+    const portalUrl = new URL(app.portals[name]);
+
+    if (portalUrl.hostname === '0.0.0.0') {
+      const {
+        port, pathname, search, hash,
+      } = portalUrl;
+      return `${this.window.location.protocol}//${this.window.location.hostname}:${port}${pathname}${search}${hash}`;
```

Comment:
```suggestion
      portalUrl.hostname = this.window.location.hostname;
```
Would this work?

Reply:
@stavros-k - is that OK to update Workloads card as well? (to not see `0.0.0.0` there as well?)

Reply:
The code is updated now to remove updated logic from `Workloads` card